pkg/app/app_test.go

```diff
@@ -217,6 +218,34 @@ func TestDeleteHostIsHandled(t *testing.T) {
 	}
 }
 
+func TestInfraConfigRequest(t *testing.T) {
+	controller := NewApp(&testInstanceManager{}, &testAccountManager{}, nil, nil, nil, "", nil, config.WebRTCConfig{STUNServers: []string{"foo.com:12345"}}, &config.Config{})
+	ts := httptest.NewServer(controller.Handler())
+	defer ts.Close()
+
+	res, _ := http.Get(ts.URL + "/v1/zones/foo/hosts/bar/infra_config")
+
+	expectedStatus := http.StatusOK
+	if res.StatusCode != expectedStatus {
+		t.Errorf("unexpected status code <<%d>>, want: %d", res.StatusCode, expectedStatus)
+	}
+
```

Comment:
remove empty lines: 232 and 242, having a continuous assert section in the test.  we aim to have 3 well defined sections in the tests: **arrange**, **act** and **assert**.

Reply:
Removed empty lines.